staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2539,8 +2539,32 @@ type KubeVirtConfiguration struct {
 	// CommonInstancetypesDeployment controls the deployment of common-instancetypes resources
 	// +nullable
 	CommonInstancetypesDeployment *CommonInstancetypesDeployment `json:"commonInstancetypesDeployment,omitempty"`
+
+	// Instancetype configuration
+	// +nullable
+	Instancetype *InstancetypeConfiguration `json:"instancetype,omitempty"`
+}
+
+type InstancetypeConfiguration struct {
+	// ReferencePolicy defines how an instance type or preference should be referenced by the VM after submission, supported values are:
+	// reference (default) - Where a copy of the original object is stashed in a ControllerRevision and referenced by the VM.
+	// expand - Where the instance type or preference are expanded into the VM during submission with references removed.
```

Comment:
Missing `expandAll`?

Reply:
https://github.com/kubevirt/kubevirt/pull/13086/commits/53c76080596df96d5155c5b44af3d1fc3da90bf1

Reply:
I don't recall `expandAll` from the design proposal. FWIW, I would vote against this complexity. Are you aware of any user that would like to differentiate between `expand` and `expandAll`?

Reply:
Yes apologies I was out last sick week and have had to implement this rather quickly given the FF deadline. I've updated the proposal earlier today with this new policy approach. The additional complexity to have an `expand` and `expandAll` policy is actually minimal and gives us some flexibility if we move ahead with this.

FWIW I've not heard of a single end user asking for any of this functionality so if that's a requirement we might want to let a few stakeholders downstream know.